summaries/GIRAFFE.md

```diff
@@ -0,0 +1,44 @@
+# GIRAFFE: Representing Scenes as Compositional Generative Neural Feature Fields
+Michael Niemeyer, Andreas Geiger
+
+## Summary
+<img src='../images/GIRAFFE_overview.PNG'>
+
+The paper presents a **generative neural feature fields** based approch for representing scenes as a composition 
+of multiple objects and a background which allows better disentanglement of scene properties like object shapes, 
+appearences and camera pose without any posed images or any other supervision. The model is able to disentangle 
+individual objects and allows rotation and translation of the individual objects as well as change in camera pose.
+
+## Contributions
+
+- A novel method for generating scenes in a controllable and photorealistic manner while training from unposed images.
+- Incorporating a compositional 3D scene representation directly into the generative model to allow more controllable 
+image synthesis and using neural rendering for contiuous scene representation for more realistic images.
+
+## Model
+
+<img src='../images/GIRAFFE_model.PNG'>
+
+The model builds on the GRAF model where a scene was modeled as a neural feature field conditioned on
+two latent vectors the **appearance code zₐ** and the **shape code zₛ** sampled randomly from a distribution. 

```

Comment:
Missing `colon` after vectors, probably? Pass the text through Grammarly if not done already.